lib/modules/filteReddit.js

```diff
@@ -493,74 +488,60 @@ function executeCustomFilters(thing) {
 	return filters.find(filter => config[filter.body.type].evaluate(thing, filter.body, config));
 }
 
-const regexRegex = /^\/(.*)\/([gim]+)?$/;
```

Comment:
This change is going to break any of the basic filters that use a regex. Might need to refactor how the post filters work to accept regex flags in order to fully implement this change